### DISCUSSION TITLE
Sky PC bug report

### DISCUSSION THREAD

**Discussion by zerobest1123** [thatskymod/Sky-CotL-Scripts#93] [General]
After I use auto wax and eden run the game block most of my friend on first two constellation

**Comment by zerobest1123** [comment]
> After I use auto wax and eden run the game block most of my friend on first two constellation

I also click on daily routne as well

**Comment by thatskymod** [comment]
Oh, that is a really serious bug, and I apologize for what happened!

A plausible explanation would be that the 'Light All' function is supposed to loop through the friends list, lighting each friend until the end of the list is reached. Then it should block visible friends and light the next portion. This is an awkward, yet natural method, rooted in the fact that Sky doesn't allow fetching the entire friend list with a single request. Of course, the block needs to be reversed once all friends have been given light.

I'll see what can be done.

**Reply by zerobest1123** [reply]
I just relise the word light all mean light friend star lol
Also the daily routine stuck at 95% on the TGC office
I have left PC run for an hours and it still stuck there so might be problem? idk

**Comment by batatajedai** [comment]
I was just using the Daily Routine to farm faster. But today when I went to run the program it is no longer working, in the past it stuck at 98% but I received the candles. Now the program only closes by itself before it even reaches 98%, would there be any solution or other program just to farm daily?   :(

**Comment by thatskymod** [comment]
A better PC mod is now available on GitHub: [ThatSkyMod by XeTrinityz](https://github.com/XeTrinityz/ThatSkyMod), updated for Sky version 0.27.1.296174. This update effectively resolves the issue.
